### ISSUE TITLE
Allow slackbot to pull from Looks 

### ISSUE THREAD

**Issue by haarthi** [looker/looker-slackbot#47]
+1 Pelikantravel

It would be great if we could issue commands directly to looks instead of adding looks to dashboards and then connecting a slack command with a dashboard.

**Comment by MaximilianR** [comment]
Now that Looks have been 'upgraded' to first-class citizens, is there even a reason to support dashboards?

Because we now have a bunch of Dashboards that have one Look and only exist for the Looker Bot...

**Comment by wilg** [comment]
Yea, it's a little weird at the moment. The reason is so you can choose which filters to expose to the Slackbot via dashboard filters.

**Comment by MaximilianR** [comment]
Thanks for the reply.

But how is choosing which filters to expose through a Dashboard substantially different from through a Look?

**Comment by wilg** [comment]
You can't choose which filters to expose on a Look currently – all the filters on the query are exposed by the Look. It's often the case that a dashboard filter will apply to only one of the many filters on an underlying tile.

**Comment by MaximilianR** [comment]
Ah yes that makes sense. 

Though if there were some way of navigating around that, it would mean we weren't creating separate dashboards for Looker only

**Comment by wilg** [comment]
Yep, totally. I get how dashboards are a bit of an ugly solution here. We're definitely looking into some ways to make it easier to set up Slack commands.
